beitragsordnung.md

```diff
@@ -18,22 +21,49 @@ Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 fo
     | Kinder (bis zum vollendeten 14. Lebensjahr) | 7,50 Euro | - |
     | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |
 
-5. Für die Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.
+Der Quartale bzw. (Halb-)Jährliche Beitrag berechnet sich entsprechend.
+
+
+2. Tagessatz
+2.1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.
+
+2.2. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.
+
+2.3 Den Mitgliedern steht die Möglichkeit einer 10er-Karte zur Verfügung. Diese ist ähnlich einen Gutschein nutzbar und eingenständig zu entwerten. Es ist Vorkasse (Bar oder Überweisung) erforderlich. Die Kosten bzw. der Wert der 10er-Karte entspricht zehn einzelnen Tagesätzen für Mitglieder gem. der Beitragsordnung.
+
+
+3. Arbeitsleistung
+3.1. Alle ordentlichen Mitglieder mit Stimmrecht ab dem vollendeten 16. Lebensjahr und bis zu vollendeten 60. Lebensjahr müssen jährlich 5 Stunden Arbeit zum Erhalt, zur Pflege und/oder Erweiterung der vereinseigenen Einrichtungen und Anlagen erbringen. 
+
+3.2 Die Arbeitsstunden können dabei aus der Aufsicht der offenen Werkstatt, Kursleiter, Repair-Café Helfer, Ausbau und Pflege der Werksttat bestehen. 
+
+3.2 Kontrolle erbrachter Arbeitsstunden obliegt dem Vorstand, dabei werden die Arbeitsstunden einzelner Mitglieder durch den Vorstand oder den Mitgliederwart schriftlich bestätigt.
+
+3.3 Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleistete Stunde 10 Euro. Der eventuell fällig werdende Betrag wird dem Mitglied schriftlich mitgeteilt. Im folgenden Monat wird der Betrag durch das Mitglied in bar, per Lastschrift oder per Überweisung beglichen.
```

Comment:
```suggestion
3.3 Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleisteter Stunde 10 Euro. Der eventuell fällig werdende Betrag wird dem Mitglied schriftlich mitgeteilt. Im folgenden Monat wird der Betrag durch das Mitglied in bar, per Lastschrift oder per Überweisung beglichen.
```

Schriftlich = per Brief. Reicht nicht zwecks Praktikabilität auch per E-Mail?

Reply:
Gem. § 126 BGB darf die schriftliche Form (Brief), doch sowieso durch die elektronische Form (E-Mail) ersetzt werden wenn nicht anders vorgeschrieben. Also ich würde vorschlagen eine Excel Liste in der Cloud zu führen und die Mitglieder müssten sich beim Vorstand melden und bekommen dann ein Bestätigung.

Reply:
Elektronische Form meint aber (gem. § 126a BGB) mit einer qualifizierten elektronischen Signatur, das macht es dann etwas komplizierter. Ich würde daher einfach aufnehmen "per E-Mail" statt "schriftlich"

Reply:
Änderst du es dann in dem Entwurf? Dann können wir diesen Punkt schließen.